src/main/java/org/folio/marccat/dao/persistence/SeeReferenceTag.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.marccat.dao.persistence;
+
+/**
+ * @author elena
+ *
+ */
+public class SeeReferenceTag extends SeeSeeAlsoReference {
+
+  /**
+   * 
+   */
+  private static final long serialVersionUID = -7888631047592313869L;
+
+  public SeeReferenceTag() {
+    super();
+  }
+
```

Comment:
Override the "equals" method in this class. Please, resolve the SonarCloud alert.

Reply:
I have added it